cookbook/map.rst.inc

```diff
@@ -143,27 +143,30 @@
   * :doc:`/cookbook/routing/redirect_trailing_slash`
   * :doc:`/cookbook/routing/extra_information`
 
-* :doc:`/cookbook/security/index`
+* :doc:`Authentication </cookbook/security/index>`
```

Comment:
Per our conversation on IRC:

`Security Authentication (Identifying/Logging in the User)`

`Security Authorization (Denying Access)`